benchmark/feldera-sql/README.md

```diff
@@ -98,5 +98,5 @@ This step has the following sub-steps:
 To run the benchmark itself, run:
 
 ```
-python3 benchmark/feldera-sql/run.py --api-url $FELDERA_API --kafka-broker $KAFKA_FROM_FELDERA
+python3 benchmark/feldera-sql/run.py --api-url $FELDERA_API -O bootstrap.servers=$KAFKA_FROM_FELDERA
```

Comment:
If you're going to delete `--kafka-broker` then you'll need to also update `benchmark/run-nexmark.sh`, which uses it.

Reply:
Might also want to update https://github.com/feldera/feldera/blob/main/scripts/bench.bash (which runs this script too)